### ISSUE TITLE
[Failing Test][sig-node] ci-kubernetes-node-kubelet-features

### ISSUE THREAD

**Issue by encodeflush** [kubernetes/kubernetes#103485]
#### Which jobs are failing:

ci-kubernetes-node-kubelet-features

#### Which test(s) are failing:

 * E2eNode Suite: [sig-node] ResourceMetricsAPI [NodeFeature:ResourceMetrics] when querying /resource/metrics should report resource usage through the resource metrics api 
 * kubetest.Node Tests [runner]

#### Since when has it been failing:

Jul 02 00:04:41 UTC+0200

#### Testgrid link:

https://testgrid.k8s.io/sig-release-master-informing#node-kubelet-features-master

#### Reason for failure:
E2eNode Suite: [sig-node] ResourceMetricsAPI [NodeFeature:ResourceMetrics] when querying /resource/metrics should report resource usage through the resource metrics api:
```
_output/local/go/src/k8s.io/kubernetes/test/e2e_node/resource_metrics_test.go:66
Timed out after 60.001s.
Expected
    <string>: KubeletMetrics
to match keys: {
[."container_start_time_seconds"[resource-metrics-7289::stats-busybox-0::busybox-container]:
	Expected
	    <string>: Sample
	to match fields: {
	.Value:
		Expected
		    <model.SampleValue>: 1.6254856299001548e+09
		to be >=
		    <int64>: 1625485589219944576
	.Timestamp:
		Expected
		    <time.Time>: 1970-01-01T00:00:00Z
		to be >=
		    <time.Time>: 2021-07-05T11:46:29.219945659Z
	}
	, ."container_start_time_seconds"[resource-metrics-7289::stats-busybox-1::busybox-container]:
	Expected
	    <string>: Sample
	to match fields: {
	.Value:
		Expected
		    <model.SampleValue>: 1.6254856296673653e+09
		to be >=
		    <int64>: 1625485589219946607
	.Timestamp:
		Expected
		    <time.Time>: 1970-01-01T00:00:00Z
		to be >=
		    <time.Time>: 2021-07-05T11:46:29.219947367Z
	}
	]
}
```
e2e.go: Node Tests :
```
error during go run /go/src/k8s.io/kubernetes/test/e2e_node/runner/remote/run_remote.go --cleanup --logtostderr --vmodule=*=4 --ssh-env=gce --results-dir=/workspace/_artifacts --project=k8s-infra-e2e-boskos-017 --zone=us-west1-b --ssh-user=prow --ssh-key=/workspace/.ssh/google_compute_engine --ginkgo-flags=--nodes=8 --focus="\[NodeFeature:.+\]" --skip="\[Flaky\]|\[Serial\]" --test_args=--kubelet-flags="--cgroups-per-qos=true --cgroup-root=/" --test-timeout=1h5m0s --image-config-file=/workspace/test-infra/jobs/e2e_node/image-config.yaml: exit status 1
```
Build Log:
```
W0705 11:48:29.734] Latency metrics for node tmp-node-e2e-0ed1f3a3-cos-81-12871-1290-12
W0705 11:48:29.734] Jul  5 11:48:28.591: INFO: Waiting up to 3m0s for all (but 0) nodes to be ready
W0705 11:48:29.734] STEP: Destroying namespace "resource-metrics-4001" for this suite.
W0705 11:48:29.734] 
W0705 11:48:29.734] 
W0705 11:48:29.734] • Failure [87.686 seconds]
W0705 11:48:29.735] [sig-node] ResourceMetricsAPI [NodeFeature:ResourceMetrics]
W0705 11:48:29.735] _output/local/go/src/k8s.io/kubernetes/test/e2e_node/framework.go:23
W0705 11:48:29.735]   when querying /resource/metrics
W0705 11:48:29.735]   _output/local/go/src/k8s.io/kubernetes/test/e2e_node/resource_metrics_test.go:45
W0705 11:48:29.735]     should report resource usage through the resource metrics api [It]
skipped 58 lines unfold_more
W0705 11:48:29.743] 
W0705 11:48:29.743] [Fail] [sig-node] ResourceMetricsAPI [NodeFeature:ResourceMetrics] when querying /resource/metrics [It] should report resource usage through the resource metrics api 
W0705 11:48:29.743] _output/local/go/src/k8s.io/kubernetes/test/e2e_node/resource_metrics_test.go:107
W0705 11:48:29.743] 
W0705 11:48:29.743] Ran 42 of 353 Specs in 214.148 seconds
W0705 11:48:29.743] FAIL! -- 41 Passed | 1 Failed | 0 Pending | 311 Skipped
W0705 11:48:29.743] 
W0705 11:48:29.743] 
W0705 11:48:29.744] Ginkgo ran 1 suite in 3m35.720324472s
W0705 11:48:29.744] Test Suite Failed
W0705 11:48:29.744] , err: exit status 1
```

#### Anything else we need to know:
/sig node

**Comment by pacoxu** [comment]
dup with #103428 and may be fixed by #103429

**Comment by ehashman** [comment]
> #### Which jobs are failing:
> 
> https://testgrid.k8s.io/sig-node-containerd#containerd-node-features (containerd)
> https://testgrid.k8s.io/sig-node-cri-o#ci-crio-cgroupv1-node-e2e-features (crio)
> #### Which test(s) are failing:
> 
> [sig-node] ResourceMetricsAPI [NodeFeature:ResourceMetrics] when querying /resource/metrics should report resource usage through the resource metrics api
> #### Since when has it been failing:
> 
> Ever since this commit was merged #102444
> #### Testgrid link:
> 
> https://testgrid.k8s.io/sig-node-containerd#containerd-node-features (containerd)
> https://testgrid.k8s.io/sig-node-cri-o#ci-crio-cgroupv1-node-e2e-features (crio)
> #### Reason for failure:
> 
> We need to see what the commit is breaking this particular NodeFeatures test on both crio and containerd.

from https://github.com/kubernetes/kubernetes/issues/103428

/triage accepted

**Comment by kzys** [comment]
Resolved? #103429 has been merged.

**Comment by saschagrunert** [comment]
Yes, this one can be closed
/close
